src/content/tutorials/fleet-management/app-platform/app-bundle/index.md

```diff
@@ -0,0 +1,193 @@
+
+![App bundle installation process](app-bundle-installation.png)
+
+When `App` custom resource for the bundle is created within the management cluster, it's first picked up by the management cluster's app operator which is called, by convention, `unique`. It lives under the protected `giantswarm` namespace, but in principle operates the same way the workload clusters app operators do. Essentially, it reconciles the `App` custom resource by creating a corresponding [`Chart` custom resource]({{< relref "/vintage/use-the-api/management-api/crd/charts.application.giantswarm.io.md" >}}) in the `giantswarm` namespace, which is then picked up by the management cluster's chart operator also living in that namespace. The chart operator then installs the app resources, which in this case are nested `App` custom resources to be created in the workload cluster namespace. Once delivered, the nested `App` custom resources are picked up by the workload cluster's app operator and installed in the well known way.
+
+<div class="feedback well">
+
+<h5>
+<i class="fa fa-help-outline"></i>
+Is it possible to install app bundles without involving the management cluster layer
+</h5>
+No, at least not in the current implementation form. The reason is, app bundles install `App` custom resources that are understood only by the management clusters. Attempt to install them directly to the workload cluster would result in an error because corresponding `App` custom resource definition isn't available there, resulting in the `kubernetes` not understanding the submitted objects.
+</div>
+
+Now, obviously in order to leverage this management cluster layer, the bundle `App` custom resource must be configured in a certain way which is slightly different in comparison to the solitary apps. Find more details in the [Technical Details paragraph](#technical-details).
+
+### Technical details {#technical-details}
+
+#### Naming the bundle's `App` resource
+
+The first, not so obvious, consequence of installing a bundle in a management cluster before its nested apps are installed in the workload cluster, is the requirement for name uniqueness. When a given app bundle is to be installed for more than one workload cluster, each `App` custom resource instance of this bundle must be given a unique name within the management cluster.
+
+It comes natural for the solitary apps, say the [`Hello World` app](https://github.com/giantswarm/hello-world-app),when one needs to install it twice in the workload cluster, both `App` custom resources instances must be named uniquely. It's natural because both live under the same namespace, but even if we could break them into separate namespaces, this uniqueness would still be demanded because both apps targets the same workload cluster, hence must be distinguished by its operators.
```

Comment:
This doesn't read well at all ...

```suggestion
It becomes natural for solitary apps, for example, the [`Hello World` app](https://github.com/giantswarm/hello-world-app), that when one needs to install it twice in the workload cluster, both `App` custom resources instances must be named uniquely. It's natural because both live under the same namespace, but even if we could break them into separate namespaces, this uniqueness would still be demanded because both apps targets the same workload cluster, hence must be distinguished by its operators.
```